src/sandbox/isolate_sandbox.cpp

```diff
@@ -165,7 +165,7 @@ void isolate_sandbox::isolate_init_child(int fd_0, int fd_1)
 
 	args[0] = isolate_binary_.c_str();
 	args[1] = "--cg";
-	args[2] = ("--box-id=" + std::to_string(id_)).c_str();
+	args[2] = strdup(("--box-id=" + std::to_string(id_)).c_str());
```

Comment:
This fix is a memory leak. Yes, subsequent exec will take care of that, but I think this deserves a comment.

Reply:
So it is not a memory leak (or a comment would not help ☺).

Maybe it would be appropriate to `free` the argument after the exec (for documentation and for the unlikely case when isolate cannot be executed). I'll do this in my next commit, also adding the `free` to *existing code* which does not contain it.